android/src/main/java/com/pdftron/pdftronflutter/FlutterDocumentActivity.java

```diff
@@ -71,6 +93,20 @@ public static void openDocument(Context packageContext, Uri fileUri, String pass
         packageContext.startActivity(intent);
     }
 
+    public static void setLeadingNavButtonIcon(String leadingNavButtonIcon) {
+        FlutterDocumentActivity documentActivity = getCurrentActivity();
+        if (documentActivity != null) {
+            PdfViewCtrlTabHostFragment pdfViewCtrlTabHostFragment = documentActivity.getPdfViewCtrlTabHostFragment();
+            if (mShowLeadingNavButton && pdfViewCtrlTabHostFragment != null
+                    && pdfViewCtrlTabHostFragment.getToolbar() != null) {
+                int res = Utils.getResourceDrawable(documentActivity.getApplicationContext(), leadingNavButtonIcon);
```

Comment:
remove getCurrentActivity
change to `pdfViewCtrlTabHostFragment.getToolbar().getContext()`